src/NinjaTools.FluentMockServer/Utils/IMockServerLogger.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using JetBrains.Annotations;
+
+namespace NinjaTools.FluentMockServer.Utils
+{
+    public interface IMockServerLogger
+    {
+        void Debug([NotNull] string message, params object[] args);
+        void Error([NotNull] string message, params object[] args);
+    }
+
+    public class NullMockServerLogger : IMockServerLogger
+    {
+        private static readonly Lazy<IMockServerLogger> _factory = new Lazy<IMockServerLogger>(() => new  NullMockServerLogger());
+
+        public static IMockServerLogger Instance => _factory.Value;
+
+        /// <inheritdoc />
+        public void Debug(string message, params object[] args)
+        {
+            // Null implementation
+        }
+
+        /// <inheritdoc />
+        public void Error(string message, params object[] args)
+        {
+            // Null implementation
+        }
+    }
+
+    public class MockServerTestLogger : IMockServerLogger
+    {
+        private static readonly Lazy<IMockServerLogger> _factory = new Lazy<IMockServerLogger>(() => new MockServerTestLogger());
+
+        public static IMockServerLogger Instance => _factory.Value;
+
+        /// <inheritdoc />
+        public void Debug(string message, [CanBeNull] params object[] args)
+        {
+            Console.WriteLine(message, args);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4885942)